rotkehlchen/chain/evm/decoding/airdrops.py

```diff
@@ -15,6 +16,7 @@ def match_airdrop_claim(
         amount: 'FVal',
         asset: 'Asset',
         counterparty: str,
+        airdrop_identifier: str,
```

Comment:
Couldn't we do a `Literal['shutter', 'cornichon', ..., 'degen']` here? Would it not make sense to have the possible literals so that type checker helps us? Would help whenever it's not dynamically read